pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/strings/InefficientEmptyStringCheckTest.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.strings;
+
+import net.sourceforge.pmd.testframework.SimpleAggregatorTst;
+
+/**
+ * @author Clément Fournier (clement.fournier@insa-rennes.fr)
+ *
+ */
+public class InefficientEmptyStringCheckTest extends SimpleAggregatorTst {
```

Comment:
This class should not be necessary, the [`StringRulesTest`](https://github.com/pmd/pmd/blob/master/pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/strings/StringsRulesTest.java#L20) already checks the tests on that xml spec.